jina/clients/__init__.py

```diff
@@ -1,8 +1,10 @@
 __copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
 __license__ = "Apache-2.0"
 
+if False:
+    import PyClient
```

Comment:
PyClient does not exist as package, the right one is `from .python import PyClient`

Reply:
Great. It's an interesting way of handling that issue.

Reply:
> PyClient does not exist as package, the right one is `from .python import PyClient`

Alright will update that in the next commit.